README.md

```diff
@@ -19,21 +19,28 @@ Please refer to https://godoc.org/github.com/centrifuge/go-substrate-rpc-client
 
 ## Contributing
 
-1. Install dependencies by running `make` followed by `make install`
-1. Run tests `make test`
-1. Lint `make lint` (you can use `make lint-fix` to automatically fix issues)
+1. Install dependencies by running `make`
+2. Build the project with `go build`
+3. Lint `make lint` (you can use `make lint-fix` to automatically fix issues)
 
-## Run tests in a Docker container against the Substrate Default Docker image
+### Testing
 
-1. Run the docker container `make test-dockerized`
+We run our tests against a Substrate Docker image. You can choose to run
+the tests within a tests-dedicated Docker container or without a container.
 
-## Run tests locally against the Substrate Default Docker image
+In either case, it's important that you run the tests against a fresh Docker Substrate
+container, as the tests are not necessarily designed to be idempotent.
```

Comment:
ohhh. can we list down the things that are not idempotent? 
We can start tackling them accordingly

Reply:
Thanks for raising this flag! This conclusion had raised during a talk I had with Miguel when the v14 PR was failing unreliably, as the first time was passing but subsequent tests were not (for the most part).

However, that is no longer the case and I have verified that I could run the tests multiple times in a row and have them passing consistently.

So I removed that bit and re-adjusted the rest to not require stopping the Substrate container.